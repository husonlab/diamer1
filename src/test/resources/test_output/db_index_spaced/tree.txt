parent id	node id	rank	label	kmers in database
	0	no rank	root	0
0	1	1 from root	rank1	162
0	2	1 from root	rank2	260
1	4	2 from root	organism2	438
1	5	2 from root	organism3	323
2	3	2 from root	organism1	87
