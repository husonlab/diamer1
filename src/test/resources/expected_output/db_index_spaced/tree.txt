parent id	node id	rank	label	kmers in database
	0	no rank	root	2
0	1	1 from root		329
0	2	1 from root		557
1	4	2 from root		887
1	5	2 from root		651
2	3	2 from root	organism1	179
